tests/wait_until_previous_builds_complete.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Copyright 2016 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DRONE_BUILD_NUM=${DRONE_BUILD_NUM:=0}
+prevBuildStatus=`drone build info vmware/vic $(( $DRONE_BUILD_NUM-8 ))`
+outArray=($prevBuildStatus)
+
+while [[ ${outArray[2]} == *"running"* ]]; do
```

Comment:
I'm weak on shell. This does re-execute the drone command to update the state, right?